### DISCUSSION TITLE
How Do We Handle Issuances Stemming from a Re-org or LLC Conversion?

### DISCUSSION THREAD

**Discussion by JSv4** [Open-Cap-Table-Coalition/Open-Cap-Format-OCF#148] [Suggestions] [answered]
**Per Issue #56, importing comments / suggestions to our static Google Doc into GitHub for discussion**

Another question/suggestion from @ashawber. If stock is issued in connection with reorgs or entity conversion, what is the best way to represent this using the existing spec? Do we potentially want to add additional fields or enms to, for example, the [StockParent Type](https://github.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/blob/main/docs/schema/types/StockParent.md)?

**Comment by JSv4** [comment, marked as answer]
In the case of LLC conversions, LLCs are explicitly out of scope, at least for the initial release, so I don't think we need to figure out how to handle that just yet. 

As for other types of conversions or re-orgs, this is a good question. I feel like we might be able to handle this with the existing spec. Could we not use [StockConversion events](https://github.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/blob/main/docs/schema/objects/transactions/conversion/StockConversion.md) to show the conversion of existing securities into whatever they're converting into?

**Reply by ashawber** [reply]
Yeah, that probably works.

**Reply by JSv4** [reply]
Cool @ashawber. This proposed approach could be worth flagging in the docs / FAQ.
